week01/insert_delete_getRandom.cpp

```diff
@@ -0,0 +1,50 @@
+//TC: O(1)
+//SC: O(2N)
+class RandomizedSet {
+public:
+    vector<int>rSet; ///randmize Set
```

Comment:
```suggestion
    vector<int>rSet; ///randomized Set
```
Please be careful with the spellings.